documentation/using_plugins.md

```diff
@@ -38,6 +38,35 @@ It is a module-based plugin available on the Puppet Forge and is installed with
 Bolt. [View the documentation on the forge](https://forge.puppet.com/puppetlabs/azure_inventory).
 
 
+### Environment variable
+
+The `env_var` plugin allows users to read values stored in environment variables and load
+them into an inventory or configuration file.
+
+#### Available fields
+
+The following fields are available to the `env_var` plugin.
+
+| Key | Description | Type | Default |
+| --- | ----------- | ---- | ------- |
+| **`var`** | The name of the environment variable to read from. | `String` | None |
```

Comment:
Just a style thing, but let's avoid using extra styling on code-phrases - backticks are usually enough. If it's not clear enough when rendered on the site, I can talk to the web team about adjusting the CSS.